include/internal/param_build_set.h

```diff
@@ -13,6 +13,17 @@
 
 # include <openssl/safestack.h>
 # include <openssl/param_build.h>
+# include "internal/cryptlib.h"
+
+typedef union {
```

Comment:
I have thrown these values into this header.. Is this ok?

Reply:
It's fine, it's an internal header.